plugins/rebalance/utils.go

```diff
@@ -0,0 +1,105 @@
+package rebalance
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/gluster/glusterd2/glusterd2/store"
+	rebalanceapi "github.com/gluster/glusterd2/plugins/rebalance/api"
+	log "github.com/sirupsen/logrus"
```

Comment:
We follow a convention for grouping imports. See Imports: section from this document: https://github.com/gluster/glusterd2/blob/master/doc/coding.md

Reply:
What needs to be changed here?

Reply:
@nbalacha, need to add an extra line 

```
"github.com/gluster/glusterd2/glusterd2/store"
rebalanceapi "github.com/gluster/glusterd2/plugins/rebalance/api"

log "github.com/sirupsen/logrus"
```

Reply:
Done